plugins/lang/python/python_loader.cc

```diff
@@ -443,9 +443,13 @@ runPythonScript(std::shared_ptr<PythonContext> titl,
     // if we're simply executing the body of the script
     try {
         if(elementToRun == PackageElement::MAIN) {
-                JML_TRACE_EXCEPTIONS(false);
-                boost::python::object obj
-                    = boost::python::exec(scriptSourceStr, pyControl.main_namespace);
+            JML_TRACE_EXCEPTIONS(false);
+            int argc = 1;
```

Comment:
@jeremybarnes
```
const char * argv[] = { "mldb-boost-python"};
```
doesn't work because
1. error: invalid conversion from ‘const char**’ to ‘char**’ [-fpermissive] PySys_SetArgv(argc, argv);
2. error: deprecated conversion from string constant to ‘char*’